src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -2,37 +2,10 @@
 
 import java.util.List;
 
-/**
- * Parameterize this service and add its implementation in a separate class.
- */
-public interface MachineService {
+public interface MachineService<T> {
+    List<T> getAll(Class<? extends T> type);
 
-    /**
-     * Return the list of machines.
-     * In the implementation of this method please use your MachineProducer implementations
-     * See that 'Class type' is not parametrized.
-     * Consider to parametrize this Class< PARAMETRIZE_ME > with specific wildcard bounding
-     *
-     * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
-     * @return the list of machines
-     */
```

Comment:
Don't delete the Javadoc.

Reply:
I thought it was for reference. And before that, they always wrote in common mistakes that we don’t need commented code